src/pe-employee-recognition-nominate/widget/pe-employee-recognition-nominate.u-client-script.js

```diff
@@ -1,3 +1,46 @@
-function EmployeeRecognitionNominateController() {
-  var c = this;
-}
+function($scope) {
+	
```

Comment:
consider using controller as syntax and named controllers

https://github.com/platform-experience/serviceportal-best-practice#named-controllers